devices/ctm.js

```diff
@@ -936,17 +961,17 @@ module.exports = [
             await reporting.illuminance(endpoint);
             await endpoint.read('msOccupancySensing', ['occupancy']);
             await reporting.occupancy(endpoint);
-            // Device enabled
-            await endpoint.read('genOnOff', [0x2201]);
+            // Relay State
+            await endpoint.read('genOnOff', [0x5001], {manufacturerCode: 0x1337});
             await endpoint.configureReporting('genOnOff', [{
-                attribute: {ID: 0x2201, type: dataType.boolean},
-                minimumReportInterval: 0,
+                attribute: {ID: 0x5001, type: dataType.boolean},
+                minimumReportInterval: 1,
                 maximumReportInterval: constants.repInterval.HOUR,
-                reportableChange: null}]);
+                reportableChange: 0}], {manufacturerCode: 0x1337});
         },
         exposes: [e.switch(), e.illuminance(), e.illuminance_lux(), e.occupancy(),
-            exposes.binary('device_enabled', ea.ALL, 'ON', 'OFF')
-                .withDescription('Turn the device on or off'),
+            exposes.binary('relay_state', ea.ALL, 'ON', 'OFF')
```

Comment:
What's the difference between `state` (from `e.switch()`) and `relay_state`?

Reply:
e.switch() now does what “device_enabled” did in the previous implementation. When off the product Is disabled. Meaning, when you walk past the motion sensor, it will not switch on the relay.
“relay_state” just switches the relay, and shows the current state of the relay.

Reply:
Isn't `device_enabled` a better name then?

Reply:
I agree. But the functionality got moved from the custom attribute it was on during development to the standard on/off cluster (last-minute change).
Is there an easy way to rename e.switch() without writing custom fz/tz?

Reply:
> But the functionality got moved from the custom attribute it was on during development to the standard on/off cluster (last-minute change).

I don't see why this influences the attribute name. Renaming `relay_state` to `device_enabled` should be possible.


> Is there an easy way to rename e.switch() without writing custom fz/tz?

Can you explain this more? I don't know what you are trying to do.

Reply:
What we did was:
move `device_enabled` from 0x2201 to 0x0000
and move `relay_state` from 0x0000 to 0x5001
So `device_enabled` is now controlled by `fz.on_off` and `tz.on_off`.
Renaming `relay_state` to `device_enabled` seems wrong.
Sorry for the mess, I added support for thit product too early.

Reply:
I see, it's better to clarify it in the description: `Turn the relay on or off` -> `Permanently turns the relay off`

Reply:
But it doesn’t turn the relay permanently off. That is what `device_enabeled/e.switch()` does.
`relay_state` just turns the relay off until next time the motions sensor Is triggered.

Reply:
But this device also provides a `state` (through `fz.on_off`), whats the difference between `state` and `relay_state`?

Reply:
So IMHO `state` should become `device_enabled` and `relay_state` should become `state`. This aligns with how every other z2m supported relay works.